microsoft-edge/webview2/concepts/multi-profile-support.md

```diff
@@ -0,0 +1,315 @@
+# Support multiple profiles under a single user data folder
+
+The WebView2 Multiple Profiles API allows you to create and manipulate user profiles to work with your WebView2 controls.  Profiles in WebView2 are conceptually similar to profiles in Microsoft Edge.  Multiple profile support enables a WebView2 app to have multiple profiles under a single user data folder.
+
+Each profile has a dedicated profile folder to save browser data, which provides separate browsing data storage for each user, such as cookies, user preference settings, and cached resources.  All the WebView2 controls that are associated with the same user profile share a single profile folder.
+
+
+#### Previous approach: Using a different user data folder for each WebView2 control
+
+Previously, without multi-profile support, to achieve data separation, a WebView2 app could use different user data folders for different WebView2 controls.  However, in that approach, you must run multiple WebView2 runtime instances (each including a browser process and a bunch of child processes), which consumed more system resources including memory, CPU footprint, and disk space.
+
+
+<!-- ====================================================================== -->
+## Specify the profile when creating a WebView2
+
+#### Create an options object that defines a profile
+
+The `CreateCoreWebView2ControllerOptions` method on `CoreWebView2Environment` creates an options object, `CoreWebView2ControllerOptions`, to provide specific information about a profile, including `ProfileName` and `IsInPrivateModeEnabled`.  Use this object to specify the target profile when creating a WebView2 control instance.
+
+##### [.NET/C#](#tab/dotnetcsharp)
+
+* [CoreWebView2Environment.CreateCoreWebView2ControllerOptions Method](/dotnet/api/microsoft.web.webview2.core.corewebview2environment.createcorewebview2controlleroptions)
+   * [CoreWebView2ControllerOptions Class](/dotnet/api/microsoft.web.webview2.core.corewebview2controlleroptions)
+
+##### [WinRT/C#](#tab/winrtcsharp)
+
+* [CoreWebView2Environment.CreateCoreWebView2ControllerOptions Method](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2environment#createcorewebview2controlleroptions)
+   * [CoreWebView2ControllerOptions Class](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2controlleroptions)
+
+##### [Win32/C++](#tab/win32cpp)
+
+* [ICoreWebView2Environment10::CreateCoreWebView2ControllerOptions method](/microsoft-edge/webview2/reference/win32/icorewebview2environment10#createcorewebview2controlleroptions)
+   * [ICoreWebView2ControllerOptions interface](/microsoft-edge/webview2/reference/win32/icorewebview2controlleroptions)
+
+---
+
+#### Create a WebView2 control that uses the profile
+
+The `CreateCoreWebView2ControllerWithOptions` method creates a WebView2 control and associates it with the profile you specified.  If the specified profile doesn't exist, a new profile will be created.
```

Comment:
```suggestion
The `CreateCoreWebView2ControllerWithOptions` method (or a `Create...Controller` overload that has an `options` parameter) creates a WebView2 control and associates it with the profile you specified.  If the specified profile doesn't exist, a new profile will be created.
```
Are the correct overloads listed below for each framework?  Should we add (parentWindow, options) in link text for .NET to highlight (consistently across frameworks) that you must use the overload which takes options?